src/main/container/Containerfile

```diff
@@ -0,0 +1,5 @@
+ARG BASEIMAGE=docker.io/library/busybox:latest
```

Comment:
What about UBI micro for this? https://catalog.redhat.com/software/containers/ubi8-micro/601a84aadd19c7786c47c8ea